less/footer.less

```diff
@@ -1,11 +1,13 @@
 // Footer Styles
 footer {
-  color: #22283B;
-  background: #f4f4f4;
+  color:  @footer-font-color;
+  background: @footer-background;
+  height:200px;
```

Comment:
You don't want to set a fixed height to anything unless it's part of your design file. Even so you, you can use padding top and bottom to give you that needed height. Always use the size of the content as its natural height. Don't add a fixed height.

Reply:
Other than that, the footer looks good.